docs/contributor/06-80-chart-config.md

```diff
@@ -0,0 +1,294 @@
+| infrastructureManager.<br>multiZoneCluster | If true, enables provisioning of clusters with nodes distributed across multiple availability zones | `false` |
+| infrastructureManager.<br>useSmallerMachineTypes | If true, provisions trial, freemium, and azure_lite clusters using smaller machine types | `false` |
+| kubeconfig.allowOrigins | Specifies which origins are allowed for Cross-Origin Resource Sharing (CORS) on the /kubeconfig endpoint | `*` |
+| kymaDashboardConfig.landscapeURL | The base URL of the Kyma Dashboard used to generate links to the web UI for Kyma runtimes | `https://dashboard.dev.kyma.cloud.sap` |
+| lifecycleManager.disabled | When disabled, the broker does not create, update, or delete the KymaCR | `false` |
+| metricsv2.enabled | If true, enables metricsv2 collection and Prometheus exposure | `False` |
+| metricsv2.operationResultFinishedOperationRetentionPeriod | Duration of retaining finished operation results in memory | `3h` |
+| metricsv2.operationResultPollingInterval | Frequency of polling for operation results | `1m` |
+| metricsv2.operationResultRetentionPeriod | Duration of retaining operation results | `1h` |
+| metricsv2.operationStatsPollingInterval | Frequency of polling for operation statistics | `1m` |
+| multipleContexts | If true, generates kubeconfig files with multiple contexts (if possible) instead of a single context | `False` |
+| profiler.memory | Enables memory profiler (true/false) | `False` |
+| quotaLimitCheck.enabled | If true, validates during provisioning that the assigned quota for the subaccount is not exceeded | `False` |
+| quotaLimitCheck.interval | The interval between requests to the Entitlements API in case of errors | `1s` |
+| quotaLimitCheck.retries | The number of retry attempts made when the Entitlements API request fails | `5` |
+| quotaWhitelistedSubaccountIds | List of subaccount IDs that have unlimited quota for Kyma runtimes. Only subaccounts listed here can provision beyond their assigned quota limits | `whitelist:` |
+| regionsSupportingMachine | Defines which machine type families are available in which regions (and optionally, zones) Restricts provisioning of listed machine types to the specified regions/zones only If a machine type is not listed, it is considered available in all regions | `` |
+| runtimeConfiguration | Defines the default KymaCR template. | `default: \|-      kyma-template: \|-        apiVersion: operator.kyma-project.io/v1beta2        kind: Kyma        metadata:          labels:            "operator.kyma-project.io/managed-by": "lifecycle-manager"          name: tbd          namespace: kcp-system        spec:          channel: fast          modules: []      additional-components: []` |
+| skrDNSProvidersValues | Contains DNS provider configuration for SKR clusters | `providers: []` |
```

Comment:
```suggestion
| skrDNSProvidersValues | Contains DNS provider configuration for Kyma clusters | `providers: []` |
```

Reply:
SKR is not an approved abbreviation, so it can't be used in the external documentation.